src/modules/compliance/src/lib/Engine.cpp

```diff
@@ -21,15 +21,16 @@
 
 namespace compliance
 {
-static constexpr const char* cModuleInfo = "{\"Name\": \"Compliance\","
-                                           "\"Description\": \"Provides functionality to audit and remediate Security Baseline policies on device\","
-                                           "\"Manufacturer\": \"Microsoft\","
-                                           "\"VersionMajor\": 2,"
-                                           "\"VersionMinor\": 0,"
-                                           "\"VersionInfo\": \"Dilithium\","
-                                           "\"Components\": [\"Compliance\"],"
-                                           "\"Lifetime\": 2,"
-                                           "\"UserAccount\": 0}";
+static constexpr const char* cModuleInfo =
+    "{\"Name\": \"Compliance\","
+    "\"Description\": \"Provides functionality to audit and remediate Security Baseline policies on device\","
+    "\"Manufacturer\": \"Microsoft\","
+    "\"VersionMajor\": 2,"
+    "\"VersionMinor\": 0,"
```

Comment:
Version major is not even 1, this module was not yet released once. Keep major 0 for now and increment minor?

Reply:
This was just a reformatting, but I set it to 0. There are no plans to maintain separate versioning from the overall repository.